pom.xml

```diff
@@ -156,6 +156,9 @@
   </dependencies>
 
   <properties>
+    <!-- Overriding version that comes from kie-parent to enable the 'npx' goal. -->
+    <version.frontend-maven-plugin>1.12.0</version.frontend-maven-plugin>
```

Comment:
@tiagobento hello, as this version is higher than what we have https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/pom.xml#L476, is there a specific reason to upgrade just in `kogito-editors-java`, instead of whole `kiegroup`?

If there is no reason I will open PR upgrading `frontend` version for whole `kiegroup` to keep our stack on latest versions. What do you think?